src/main/java/org/folio/service/caches/MappingParametersCache.java

```diff
@@ -87,8 +87,9 @@ private CompletableFuture<MappingParameters> loadMappingParameters(OkapiConnecti
       .toCompletableFuture()
       .thenCompose(httpResponse -> {
         if (httpResponse.getResponse().statusCode() == HttpStatus.SC_OK) {
-          LOGGER.info("loadMappingParameters:: The first chunk of organizations was loaded for cache, tenantId '{}'", tenantId);
           OrganizationCollection orgCollection = Json.decodeValue(httpResponse.getJson().encode(), OrganizationCollection.class);
+          LOGGER.info("loadMappingParameters:: The first chunk of organizations was loaded for cache, tenantId '{}', organizations '{}'",
```

Comment:
maybe log the size of the chunk as well? So that we had an indication in log the the limit was changed/or not

Reply:
oh, or orgCollection.getOrganizations().size() is the size of the chunk?

Reply:
thanks Kate, that would be useful, what do you think?

![image](https://github.com/folio-org/mod-orders/assets/138673581/abe44dff-85f3-4d62-baa3-35ad43ff5029)

Reply:
> 

it is the size of the first chunk but it may differ from the actual `settingsLimit` (chunk size) value